app/Helpers/helpers.php

```diff
@@ -67,28 +71,28 @@ function translate($value)
 {
     switch ($value) {
         case 'waiting':
-            return __('settings.book.waiting');
-            break;
+        return __('settings.book.waiting');
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.ControlStructures.SwitchDeclaration.BreakIndent</br>**Severity**: error</br>**File**: app/Helpers/helpers.php</br>**Line**: 74</br>Terminating statement must be indented to the same level as the CASE body